lib/register.js

```diff
@@ -0,0 +1,11 @@
+const addHook = require('pirates').addHook
+
+const pwd = process.cwd()
+const jiti = require('.')(pwd)
+
+addHook(
+  function (code, filename) {
```

Comment:
Any reason not using arrow return? (also we can rename first param to source :P)

Reply:
didn't want it to look too modern :stuck_out_tongue_winking_eye: - will use arrow